desktop/libs/indexer/src/indexer/file_format.py

```diff
@@ -612,6 +603,15 @@ def _guess_fields(self, sample):
     return fields
 
 
+class XLSXFormat(CSVFormat):
+  _name = "excel"
+  _description = _("Excel File")
+  _args = [
+    CheckboxArgument("hasHeader", "Has Header")
```

Comment:
Wont we be needing these args here also just like in CSVFormat?
https://github.com/cloudera/hue/blob/a6df36415f5cb2f9f572309e96c475fd0b893ba9/desktop/libs/indexer/src/indexer/file_format.py#L311-L313

Or since Excel is converted to CSV, then thats why we don't need it particularly for Excel?

Reply:
Yes, as they are fixed, when we convert excel to CSV so don't need to show these params.